samples/ios-unified/AppDelegate.cs

```diff
@@ -22,6 +22,7 @@ public class AppDelegate : UIApplicationDelegate
 	{
 		private InboxDelegate inboxDelegate;
 		private PushHandler pushHandler;
+		private RegistrationDelegate registrationDelegate;
```

Comment:
I'll remove them when I figure out why the strong wrapping isn't working. I wanted it to be in a working state.